CMakeLists.txt

```diff
@@ -95,7 +95,7 @@ endif()
 
 enable_testing()
 add_subdirectory( test )
-add_subdirectory( benchmark )
+#add_subdirectory( benchmark )
```

Comment:
I thought you changed this to exclude

Reply:
Good catch. I had this commented in two branches originally, so the catchup from `main` did not remove the comment. Fixed.